gt/main.c

```diff
@@ -1436,6 +1436,19 @@ process_cmds_from_mailbox(struct gt_instance *instance,
 	}
 }
 
+static inline void
+prefetch0_128_bytes(struct rte_mbuf *pkt)
```

Comment:
The parameter should be `void *pointer`. The current type and parameter are not a good fit.